CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ Summary:
 -   Raise a CopierAnswersInterrupt instead of a bare KeyboardInterrupt to
     provide callers with additional context - such as the partially completed
     AnswersMap.
+-   New property on Worker, and run_* variants: `user_defaults` allows
```

Comment:
I'd simplify this with: "support for user defaults"

Reply:
Met halfway with: Support for `user_defaults`, which precedence over template defaults.

Reply:
The provided tests are OK 👍